great_expectations_provider/__init__.py

```diff
@@ -9,4 +9,11 @@ def get_provider_info():
         "name": "Great Expectations Provider",
         "description": "An Apache Airflow provider for Great Expectations.",
         "versions": [__version__],
+        "connection-types": [
+            {
+                "connection-type": "gx_cloud",
+                "hook-class-name": "great_expectations_provider.hooks.gx_cloud:GXCloudConnection",
```

Comment:
```suggestion
                "hook-class-name": "great_expectations_provider.hooks.gx_cloud.GXCloudConnection",
```

Reply:
Once we refactor GXCloudConnection to GXCloudHook, we would also need to change the class-name here.